### PR TITLE
Add options to _cl_int_plot_top_losses

### PR DESCRIPTION
**Pull request by gkaissis**
The current version of `plot_top_losses` does not include an option to overlay a perceptually uniform colour map (important for e.g. colourblind users), to adjust the alpha of the overlay or to hide the text (for instance if wanting to save a figure for a publication). This PR adds this functionality. 
Also, it capitalises the figure title to aid in readability after the slashes.

### CODE REVIEW

File: fastai/vision/learner.py [hunk 1]
```diff
@@ -37,7 +37,7 @@ def _alexnet_split(m:nn.Module): return (m[0][0][6],m[1])
 
     models.densenet121:{**_densenet_meta}, models.densenet169:{**_densenet_meta},
     models.densenet201:{**_densenet_meta}, models.densenet161:{**_densenet_meta},
-    models.vgg11_bn:{**_vgg_meta}, models.vgg13_bn:{**_vgg_meta}, models.vgg16_bn:{**_vgg_meta}, models.vgg19_bn:{**_vgg_meta},
+    models.vgg16_bn:{**_vgg_meta}, models.vgg19_bn:{**_vgg_meta},
```
Comment (sgugger) [inline fastai/vision/learner.py]: Be careful, you are removing functionality here that's not linked to your PR.

File: fastai/vision/learner.py [hunk 2]
```diff
@@ -170,17 +170,20 @@ def _cl_int_plot_top_losses(self, k, largest=True, figsize=(12,12), heatmap:bool
...
     rows = math.ceil(k/cols)
     fig,axes = plt.subplots(rows, cols, figsize=figsize)
-    fig.suptitle('prediction/actual/loss/probability', weight='bold', size=14)
     for i,idx in enumerate(tl_idx):
         im,cl = self.data.dl(self.ds_type).dataset[idx]
         cl = int(cl)
-        im.show(ax=axes.flat[i], title=
-            f'{classes[self.pred_class[idx]]}/{classes[cl]} / {self.losses[idx]:.2f} / {self.preds[idx][cl]:.2f}')
+        if show_text:
+            fig.suptitle('Prediction/Actual/Loss/Probability', weight='bold', size=14)
```
Comment (sgugger) [inline fastai/vision/learner.py]: This should only be executed once before the loop.

File: fastai/vision/learner.py [hunk 3]
```diff
@@ -160,7 +160,7 @@ def _cl_int_gradcam(self, idx, heatmap_thresh:int=16, image:bool=True):
 
 ClassificationInterpretation.GradCAM =_cl_int_gradcam
 
-def _cl_int_plot_top_losses(self, k, largest=True, figsize=(12,12), heatmap:bool=False, heatmap_thresh:int=16,
+def _cl_int_plot_top_losses(self, k, largest=True, figsize=(12,12), heatmap:bool=False, heatmap_thresh:int=16, alpha:float=0.6, cmap:str="magma", show_text:bool=True,
```
Comment (sgugger) [inline fastai/vision/learner.py]: Can you put the extra args on the second line to make it easier to browse the code (recommended max char width is 120/140 I believe)

File: fastai/vision/learner.py (hunk 1 above)
Comment (gkaissis) [reply fastai/vision/learner.py]: Sure, will fix

File: fastai/vision/learner.py (hunk 2 above)
Comment (gkaissis) [reply fastai/vision/learner.py]: Won't this raise an exception because idx is referenced before assignment?

File: fastai/vision/learner.py (hunk 3 above)
Comment (gkaissis) [reply fastai/vision/learner.py]: Sure

File: fastai/vision/learner.py (hunk 2 above)
Comment (sgugger) [reply fastai/vision/learner.py]: I was talking of the line `fig.suptitle`. There is no `idx` in this one.

**Comment by gkaissis** [comment]
Something messed up my fork, closing this PR and reopening for avoiding mistakes.

**Pull request closed without merging**